File: BlogContent.Core/Enums/UserStatus.cs
namespace BlogContent.Core.Enums
{
    public enum UserStatus
    {
        PendingEmailConfirmation, // Ожидает подтверждения почты
        Active,     // Обычный пользователь
        Banned,     // Заблокированный пользователь
        Admin       // Администратор
    }
}

File: blogplatform-frontend/src/utils/userStatus.js
const STATUS_LABELS = {
  Active: 'Проверен',
  Admin: 'Администратор',
  Banned: 'Блокировка'
};

export function getUserStatusLabel(status) {
  if (!status) {
    return '—';
  }
  return STATUS_LABELS[status] ?? status;
}

export function isUserBanned(status) {
  return status === 'Banned';
}
